replica/replica.go

```diff
@@ -255,6 +259,9 @@ func (r *Replica) insertBackingFile() {
 	d := disk{Name: r.info.BackingFile.Name}
 	r.activeDiskData = append([]*disk{{}, &d}, r.activeDiskData[1:]...)
 	r.volume.files = append([]types.DiffDisk{nil, r.info.BackingFile.Disk}, r.volume.files[1:]...)
+	r.volume.ReadOnlyIndx = append(r.volume.ReadOnlyIndx, false)
+	r.diskList = append(r.diskList, r.info.BackingFile.Name)
+	r.volume.UserCreatedSnap = append([]bool{false, false}, r.volume.UserCreatedSnap[1:]...)
```

Comment:
here, the append should be reverse

Reply:
It is sync with r.volume.files. Although this feature is not being used currently.